cockroachdb/src/jepsen/cockroach/multiregister.clj

```diff
@@ -0,0 +1,131 @@
+(ns jepsen.cockroach.multiregister
+  "Multiple atomic registers test
+
+  Splits registers up into different tables to make sure they fall in
+  different ranges"
+  (:refer-clojure :exclude [test])
+  (:require [jepsen [cockroach :as cockroach]
+             [client :as client]
+             [checker :as checker]
+             [generator :as gen]
+             [reconnect :as rc]
+             [independent :as independent]
+             [util :as util]]
+            [jepsen.checker.timeline :as timeline]
+            [jepsen.cockroach.client :as c]
+            [jepsen.cockroach.nemesis :as cln]
+            [clojure.java.jdbc :as j]
+            [clojure.tools.logging :refer :all]
+            [knossos.model :as model]))
+
+(def reg-count 5)
+(def reg-range (range reg-count))
+
+(def table-prefix "String prepended to all table names." "register_")
+(defn id->table
+  "Turns an id into a table name string"
+  [id]
+  (str table-prefix id))
+(def table-names (map id->table reg-range))
+
+(defn r
+  "Read a random subset of keys."
+  [_ _]
+  ;; Reading a random subset of keys after writing a random subset of
```

Comment:
Clarify this comment: is it describing what the code does now or a pitfall avoided? Does this mean that the test currently fails sometimes for this reason?

Reply:
I haven't seen the code fail yet for this reason, but I'm hoping to to confirm that things are working as expected. I'll address this comment before merging.